pkg/cmd/docgen/diagrams.go

```diff
@@ -319,24 +319,17 @@ func runRR(r io.Reader, railroadJar string) ([]byte, error) {
 }
 
 var specs = []stmtSpec{
-	// TODO(mjibson): improve SET filtering
-	// TODO(mjibson): improve SELECT display
 	{
 		name:   "add_column",
 		stmt:   "alter_onetable_stmt",
-		inline: []string{"alter_table_cmds", "alter_table_cmd", "column_def"},
-		match:  []*regexp.Regexp{regexp.MustCompile("'ADD' ('COLUMN')? ?('IF' 'NOT' 'EXISTS')? ?column_name")},
+		inline: []string{"alter_table_cmds", "alter_table_cmd", "column_def", "col_qual_list"},
+		regreplace: map[string]string{
```

Comment:
This is great, but I do wonder whether we should just hard-code (straight-up replace of the bnf) when such complex intervention is needed?

Reply:
That was my first thought too. But then I examined the diagrams and noticed various small errors in the hard-coded diagrams. For example, the current add column diagram shows that the `col_qualification` term is optional with at most one entry. The newly generated one from this PR shows it as optional with any number of entries. There is another example in one of these in this PR where the current docs don't correctly show one of the words as optional when it is.

However, I also recognize that we would enjoy spending less time on this stuff, and so maybe we should just hard-code more stuff. I'm not opposed to that either, even if there are small errors. There's already a few things here that are hard coded, and that's great. Your choice about how to proceed in the future.